src/core/client/framework/lib/errors/unknownServerError.ts

```diff
@@ -21,6 +22,17 @@ export default class UnknownServerError extends Error {
       Error.captureStackTrace(this, UnknownServerError);
     }
 
+    const traceID: string | null | undefined = (error as any).traceID as string;
+    const traceIDs: string[] | null | undefined = (error as any)
+      .traceIDs as string[];
```

Comment:
As suggested `traceIDs` doesn't need to be handled, as we can remove `MultiError` class.

Reply:
Please see previous comment for why I created MultiError.